config/processCcdWithFakes.py

```diff
@@ -0,0 +1,31 @@
+"""
+Subaru-specific overrides for ProcessCcdWithFakesTask (applied before SuprimeCam- and HSC-specific overrides).
+"""
+import os.path
+
+from lsst.utils import getPackageDir
+from lsst.meas.algorithms import ColorLimit
+
+ObsConfigDir = os.path.join(getPackageDir("obs_subaru"), "config")
+
+bgFile = os.path.join(ObsConfigDir, "background.py")
+
+# Cosmic rays and background estimation
+config.detection.background.load(bgFile)
+
+# Set to match defaults curretnly used in HSC production runs (e.g. S15B)
```

Comment:
Typo (curretnly) and maybe mention that this is copied from obs_subaru/config/charImage.py

Reply:
I've added it to the comment at the top and fixed the typo.